guides/partials.md

```diff
@@ -1,29 +1,38 @@
 ---
-title: DustJS | Getting Started
+title: Dust.js | Partials
 layout: guides
 permalink: /guides/partials/
 ---
 
-## Partials
+# Partials
```

Comment:
Tonya has made changes to this file as well, so get ready for a merge conflict...